database/models.py

```diff
@@ -1,6 +1,7 @@
-from .db import db
 import datetime
 from utils.utils import objectToJson
+from .db import db
+
```

Comment:
Use imports like from
```suggestion
database.db import db
```